_templates/layout.html

```diff
@@ -30,6 +29,24 @@
         ePUB</a>
     </p>
   </li>
+  <li class="color-mode">
+    <p>
+      {{ _('Color') }}:
+      <a href="#" 
+        id="light-mode" 
+        @click.prevent="darkMode = 'light'; localStorage.setItem('darkMode', 'light')"
+        :class="{ 'font-bold': darkMode === 'light' }">
+        {{ _('Light') }}
+      </a>
+      /
+      <a href="#" 
+        id="dark-mode" 
+        @click.prevent="darkMode = 'dark'; localStorage.setItem('darkMode', 'dark')"
+        :class="{ 'font-bold': darkMode === 'dark' }">
+        {{ _('Dark') }}
+      </a>
+    </p>
+  </li>
```

Comment:
There's a light/dark mode button built in the awesome sphinx theme, but I had disabled it, I am going to put it back (will just need to delete `header.html` and remove this.